src/MCPClient/lib/clientScripts/jsonMetadataToCSV.py

```diff
@@ -88,7 +88,7 @@ def object_to_row(row, headers):
     """
     def sort_row(keyvalue):
         return headers.index(keyvalue[0])
-    row = sorted(row.items(), key=sort_row)
+    row = sorted(list(row.items()), key=sort_row)
```

Comment:
Is it necessary to call ``list`` here? ``sorted`` can handle any iterable as its first argument, can't it?